### PR TITLE
Feature/tests

### PR DESCRIPTION
**Pull request by KartaviK**
Fix fetching `language` attribute for panels
Fix condition for street type in response generating

### CODE REVIEW

**Review by sashabeton — APPROVED** [review]

File: tests/Unit/Panels/StreetsPanelTest.php [hunk 1]
```diff
@@ -2,27 +2,92 @@
...
+
     /** @var StreetsPanel */
-    protected $fakeStreetsPanel;
+    protected $panel;
+
+    protected function setUp()
+    {
+        parent::setUp();
+
+        $this->expectedRules = [
```
Comment (Horat1us) [inline tests/Unit/Panels/StreetsPanelTest.php]: Rules should not be checked as array equal. Rules should be checked using validation.
So, you need to fill attribute and then run ->validate(['attributeName']) with some error/success expectation.

**Review by Horat1us — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+$module = [
+    'controllerMap' => [
+        'autocomplete' => [
+            'class' => \Wearesho\GoogleAutocomplete\Yii\Controller::class,
+        ],
+    ]
+];
+```
+
+## Authors
```
Comment (Horat1us) [inline README.md]: Contributors section

File: README.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+Add to your `api/config/main.php`:
+
+```php
+<?php
+
+$module = [
+    'controllerMap' => [
+        'autocomplete' => [
+            'class' => \Wearesho\GoogleAutocomplete\Yii\Controller::class,
```
Comment (Horat1us) [inline README.md]: Example should have `use Wearesho\GoogleAutocomplete` in header.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
+# Yii2 Google autocomplete api integrate
```
Comment (Horat1us) [inline README.md]: Missing `Installation` section

**Review by Horat1us — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (Horat1us) [inline README.md]: Titles have to be first-character uppercased in english

**Review by Horat1us — CHANGES_REQUESTED** [review]

**Pull request merged**